source/documentation/developer-setup.html.md.erb

```diff
@@ -40,18 +40,16 @@ You must subscribe each application you register to CTC Guarantee Balance API v1
 
 ## API access control
 
-CTC Guarantee Balance API v1.0 is accessible only to applications that have been placed in the allow list for that API. If you cannot subscribe to or access the API on the Developer Hub, please check with the SDS Team if your `ApplicationId` needs to be added to the allow list for that API.
+CTC Guarantee Balance API v1.0 is a public API, so you should be able to gain access to it automatically and subscribe to it easily.
```

Comment:
I'd beck going back to check with SDST on this point. I think that's true for sandbox, but production requires going through the production checklist -- so worth checking exactly what the situation is here.

Reply:
I see your point.

In review comments, SDST said:
>In the Developer hub section in the API Access control heading section, the allow list for private APIs only and as CTC Guarantee balance is a public API, the Software developer should be able to self-subscribe and gain access automatically so SDST wouldn’t be able to get involved if there’s a problem with a version on the API not being visible. This will need to be reviewed again for Phase 5.
Again similar to the above bullet point in the API versioning heading section that it they can self-subscribe and that SDTS wouldn’t be able to get involved as allow listing is for private APIs only.

For now, let's go with what they said but I will mention your point when inviting them to review the changes in QA.